### PR TITLE
mcobbett 1395 launch jvm windows

### PR DESCRIPTION
**Pull request by techcobweb**
- [x] Roughly working on Windows. 
- [x] RAS path overridden only if it's not already set by a user
- [x] Local run prefix only set if it's not already set by a user
- [x] Better error messages in GALxxxE: format.
- [x] More validation of input obr and class parameters

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

File: test-locally.sh [hunk 1]
```diff
@@ -188,6 +188,61 @@ ${GALASACTL} runs submit local \
...
+--class dev.galasa.example.banking.account/dev.galasa.example.banking.account.TestFailingAccount \
+--class dev.galasa.example.banking.payee/dev.galasa.example.banking.payee.TestPayee \
+--class dev.galasa.example.banking.payee/dev.galasa.example.banking.payee.TestPayeeExtended \
+--class dev.galasa.example.banking.account/dev.galasa.example.banking.account.TestAccount \
+--class dev.galasa.example.banking.account/dev.galasa.example.banking.account.TestAccountExtended \
+--class dev.galasa.example.banking.account/dev.galasa.example.banking.account.TestFailingAccount \
+--class dev.galasa.example.banking.payee/dev.galasa.example.banking.payee.TestPayee \
+--class dev.galasa.example.banking.payee/dev.galasa.example.banking.payee.TestPayeeExtended \
+--class dev.galasa.example.banking.account/dev.galasa.example.banking.account.TestAccount \
+--class dev.galasa.example.banking.account/dev.galasa.example.banking.account.TestAccountExtended \
```
Comment (AaronJhaj) [inline test-locally.sh:232]: Mistake?

Comment (techcobweb) [reply test-locally.sh:232]: Yep. Mistake. Fixed.

**Review by AaronJhaj — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**